### PR TITLE
[merged] doc: commctl config update for multiple endpoints.

### PR DESCRIPTION
**Pull request by ashcrow**
commctl can now handle multiple hosts. This change updates the
documentation to show how to use multiple endpoints so that if one
endpoint is unreachable the next is tried.

### CODE REVIEW

File: doc/examples/commctl_config_multihost.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.. code-block:: json
+
+   {
+       "username": "a",
+       "endpoint": [
+           "http://127.0.0.1:8080",
+           "http://192.168.122.100:8080",
+           "http://10.1.1.1:8080"]
+   }
```
Comment (mbarnes) [inline doc/examples/commctl_config_multihost.rst]: Commissaire's default port is 8000.  Other examples in the docs show port 8080 being used for Kubernetes.  We should be consistent; that actually bit me a little earlier.

File: doc/operations.rst [hunk 2]
```diff
@@ -11,20 +11,26 @@ be used this document will show examples using *commctl* as well as *curl*.
...
-   {
-       "username": "a",
-       "endpoint": "http://127.0.0.1:8080"
-   }
+The password may be stored in the configuration file as well.
 
-.. note::
+.. warning::
+
+   The configuration file is plain text. If you choose to keep a password in the file make sure to keep the file locked down.
```
Comment (mbarnes) [inline doc/operations.rst]: Maybe clarify: keep the file *permissions* lock down

**Comment by ashcrow** [comment]
:arrow_up:

**Comment by mbarnes** [comment]
Looks good.  :+1: 

@cgwalters-bot r+

**Pull request closed without merging**